actionview/test/fixtures/mascot.rb

```diff
@@ -1,5 +1,5 @@
 # frozen_string_literal: true
 
 class Mascot < ActiveRecord::Base
-  belongs_to :company
+  belongs_to :company, required: true
```

Comment:
`:required` is  set to true by default and is [deprecated.](https://edgeapi.rubyonrails.org/classes/ActiveRecord/Associations/ClassMethods.html#method-i-belongs_to)
```suggestion
  belongs_to :company
```

Reply:
I’ll set it to optional: false. I don’t know why but if I remove it the tests fail